extensions/amp-story-panning-media/validator-amp-story-panning-media.protoascii

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-story-panning-media
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-story-panning-media"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-story-panning-media>
+  html_format: AMP
+  tag_name: "AMP-STORY-PANNING-MEDIA"
+  requires_extension: "amp-story-panning-media"
+  mandatory_ancestor: "AMP-STORY"
```

Comment:
I think this mandatory_ancestor can be removed since the grid layer is always a child of `AMP-STORY`